cms/jinja2/templates/pages/cookies.html

```diff
@@ -0,0 +1,192 @@
+{% extends "templates/base.html" %}
+{% from "components/radios/_macro.njk" import onsRadios %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/details/_macro.njk" import onsDetails %}
+{% from "components/table/_macro.njk" import onsTable %}
+
+{% block main %}
+    <div class="ons-page__body">
+        <h1 class="ons-u-fs-2xl">{{ _("Cookies on ONS.GOV.UK") }}</h1>
+
+        {# fmt:off #}
+        {% call onsPanel({
+                "classes": "ons-u-mb-l ons-u-d-no ons-cookies-confirmation-message",
+                "variant": "success",
+                "iconType": "check",
+                "iconSize": "xl"
+        }) %}
+            <h2>{{ _("Your cookie settings have been saved") }}</h2>
+            <p class="ons-u-mb-no">
+                {{ _("Some parts of this website may use additional cookies and will have their own cookie policy and banner.") }}
+            </p>
+            <a class="ons-u-mt-s ons-u-dib ons-js-return-link" href="#0">{{ _("Return to previous page") }}</a>
+        {% endcall %}
+        {# fmt:on #}
+
+        <p>{{ _("Cookies are files saved on your phone, tablet or computer when you visit a website.") }}</p>
+        <p>{{ _("We use cookies to store information about how you use the ONS website, such as the pages you visit.") }}</p>
+
+        <h2>{{ _("Cookie settings") }}</h2>
+
+        <div class="ons-u-db-no-js_enabled">
+            <p>
+                {{ _("We use JavaScript to set our cookies. Javascript is not running on your browser so you cannot change
+                your settings.") }}
+            </p>
+            <p>{{ _("You can try:") }}</p>{{ _("Cookie settings") }}
```

Comment:
Looks like a copy and paste mistake.

```suggestion
            <p>{{ _("You can try:") }}</p>
```